projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DownloadKey.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import { VdkOption } from '../vdkOptions/vdk_options';
+import VDKTextInput from './VdkTextInput';
+import { Dialog, showDialog } from '@jupyterlab/apputils';
+import { jobRequest } from '../serverRequests';
+import { IJobPathProp } from './props';
+import { jobData } from '../jobData';
+import { DOWNLOAD_KEY_BUTTON_LABEL } from '../utils';
+
+export default class DownloadKeyDialog extends Component<IJobPathProp> {
+  /**
+   * Returns a React component for rendering a download menu.
+   *
+   * @param props - component properties
+   * @returns React component
+   */
+  constructor(props: IJobPathProp) {
+    super(props);
+  }
+  /**
+   * Renders a dialog for downloading a data job key.
+   *
+   * @returns React element
+   */
+  render(): React.ReactElement {
+    return (
+      <>
+        <VDKTextInput
+          option={VdkOption.NAME}
+          value="default-name"
+          label="Job Name:"
+        ></VDKTextInput>
+        <VDKTextInput
+          option={VdkOption.TEAM}
+          value="default-team"
+          label="Job Team:"
+        ></VDKTextInput>
+        <VDKTextInput
+          option={VdkOption.PATH}
+          value={this.props.jobPath}
```

Comment:
The key should not be downloaded inside the job directory but alongside it. 

```
/root
|-- job-dir/
|   |-- config.ini
|   |-- ...
|-- job-key

 ```

Reply:
changed this to the parent directory

Reply:
@antoniivanov  if they save it somewhere else will it use it? 
Or does it need to be saved exactly there? 
maybe they should not be able to choose where to save it to. 
Make it simpler for customers.

Reply:
If they save it somewhere else , they could configure it (using [config described here](https://github.com/vmware/versatile-data-kit/tree/main/projects/vdk-plugins/vdk-kerberos-auth#configuration) which nobody ever does . By large vdk download-key CLI downloads it in the right location . If for some reason it is not they would move it.